cmake/fw_zip.cmake

```diff
@@ -0,0 +1,36 @@
+
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+function(generate_dfu_zip zip_out bin_file_list deps)
+  print(zip_out)
+  print(bin_file_list)
+  print(deps)
+  print(PYTHON_EXECUTABLE)
+  print(NRF_DIR)
```

Comment:
Debug prints? :)

Reply:
hahaha, yes indeed. At least I leave them easy to find